constants/statusCodes.go

```diff
@@ -0,0 +1,71 @@
+package constants
+
+const (
+	StatusContinue           = 100 // RFC 9110, 15.2.1
+	StatusSwitchingProtocols = 101 // RFC 9110, 15.2.2
+	StatusProcessing         = 102 // RFC 2518, 10.1
+	StatusEarlyHints         = 103 // RFC 8297
+
+	StatusOK                          = 200 // RFC 9110, 15.3.1
+	StatusCreated                     = 201 // RFC 9110, 15.3.2
+	StatusAccepted                    = 202 // RFC 9110, 15.3.3
+	StatusNonAuthoritativeInformation = 203 // RFC 9110, 15.3.4
+	StatusNoContent                   = 204 // RFC 9110, 15.3.5
+	StatusResetContent                = 205 // RFC 9110, 15.3.6
+	StatusPartialContent              = 206 // RFC 9110, 15.3.7
+	StatusMultiStatus                 = 207 // RFC 4918, 11.1
```

Comment:
We can have the map like this

And also this map should be declared in `constants/constants.go` as a constant


```
HTTPStatusCodes := map[int]string{
		100: "Continue",
		101: "Switching Protocols",
		102: "Processing",
		200: "OK",
		201: "Created",
		202: "Accepted",
		301: "Moved Permanently",
		302: "Found",
		307: "Temporary Redirect",
		308: "Permanent Redirect",
		400: "Bad Request",
		401: "Unauthorized",
		402: "Payment Required",
		403: "Forbidden",
		404: "Not Found",
		405: "Method Not Allowed",
		406: "Not Acceptable",
		408: "Request Timeout",
		415: "Unsupported Media Type",
		429: "Too Many Requests",
		500: "Internal Server Error",
		501: "Not Implemented",
		502: "Bad Gateway",
		503: "Service Unavailable",
		504: "Gateway Timeout",
		505: "HTTP Version Not Supported"
	}
```